src/driver/postgres/PostgresDriver.ts

```diff
@@ -727,12 +727,14 @@ export class PostgresDriver implements Driver {
             (
                 columnMetadata.type === "enum"
                 || columnMetadata.type === "simple-enum"
+                || columnMetadata.isArray
```

Comment:
hmmm what if `column.type` is a `string` and `isArray` set to `true`? It shouldn't go into this block isn't it?